apps/igloo-kit-cli/src/cli/mod.rs

```diff
@@ -81,20 +81,33 @@ impl CommandTerminal {
 }
 
 async fn top_command_handler(term: &mut CommandTerminal, config: Config, commands: &Option<Commands>, debug: bool) {
-
-    if (!config.features.coming_soon_wall) {
+    let clickhouse_config = ClickhouseConfig {
+        db_name: "local".to_string(),
+        user: "panda".to_string(),
+        password: "pandapass".to_string(),
+        host: "localhost".to_string(),
+        host_port: 18123,
+        postgres_port: 9005,
+        kafka_port: 9092,
+        cluster_network: PANDA_NETWORK.to_owned(),
+    };
+
+    if !config.features.coming_soon_wall {
         match commands {
             Some(Commands::Init {}) => {
                 routines::initialize_project(term);
             }
             Some(Commands::Dev{}) => {
-            routines::start_containers(term);
+            routines::start_containers(term, clickhouse_config.clone());
             infrastructure::setup::validate::validate_red_panda_cluster(term, debug);
-            routines::start_development_mode(term).await;      
+            routines::start_development_mode(term, clickhouse_config.clone()).await;      
         }
         Some(Commands::Update{}) => {
             // This command may not be needed if we have incredible automation
-            todo!("Will update the project's underlying infrascructure based on any added objects")
+            // todo!("Will update the project's underlying infrascructure based on any added objects")
+            let path = "/Users/timdelisle/Dev/igloo-stack/apps/igloo-kit-cli/tests/psl/simple.prisma";
```

Comment:
Hum we should probably remove this now, no?

Reply:
fixed